js/main.js

```diff
@@ -0,0 +1,50 @@
+var launchList;
+var xhrLaunches = new XMLHttpRequest();
+xhrLaunches.open('GET', 'https://ll.thespacedevs.com/2.0.0/launch/upcoming/?format=json');
+xhrLaunches.responseType = 'json';
+xhrLaunches.addEventListener('load', function () {
+  launchList = xhrLaunches.response;
+  renderUpcomingLaunches();
+});
+xhrLaunches.send();
+
+function renderUpcomingLaunches() {
```

Comment:
Rather than making this function append all the elements directly to the DOM, I would break this up a bit.
1. Make this function accept a parameter of an array of launch data
2. Make a separate function to render a single launch, and have it return a DOM tree for that entry
3. In your loop, call that function for each entry, and you can append each of those entries to your `$newSection` container
4. Return the DOM tree of all launches rather than appending directly.
5. Append the return where this function is being called in your `load` event handler